utils/sumo/patches/020-python.patch

```diff
@@ -0,0 +1,11 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -197,7 +197,7 @@ if (SKBUILD OR ${CMAKE_VERSION} VERSION_
+     find_package(PythonInterp REQUIRED)
+     find_package(PythonLibs)
+ else()
+-    find_package(Python REQUIRED COMPONENTS Interpreter Development)
++    find_package(Python COMPONENTS Interpreter Development)
```

Comment:
Would be good, if this patch have header - who authored it, what is the patch title, description, so it can be applied with ``git am``. Also, this is something what should be discussed with upstream, otherwise we will have this patch for ages. :-(